lib/logzio-nodejs.js

```diff
@@ -184,7 +185,7 @@ class LogzioLogger {
 
         if (this.addTimestampWithNanoSecs) {
             const time = process.hrtime();
-            msg['@timestamp_nano'] = msg['@timestamp_nano'] || [now, time[0].toString(), time[1].toString()].join('-');
+            msg['@timestamp_nano'] = msg['@timestamp_nano'] || [now, this._padNumberWithZeros(time[1].toString())].join('-');
```

Comment:
Why not just
```suggestion
            msg['@timestamp_nano'] = msg['@timestamp_nano'] || [now, time[1].toString().padStart(nanoSecDigit, '0')].join('-');
```

Reply:
Divided to 3 tests: 
default case of no nanosec-timestamp
add nano-sec timestamp option
valid nano-sec timestamp
https://github.com/logzio/logzio-nodejs/pull/79/commits/c326c92c109a70e751218df3a9f8798ba2f03399
@AmirTugi

Reply:
fixed